packages/eas-build-job/src/common.ts

```diff
@@ -5,6 +5,7 @@ import { BuildPhase, BuildPhaseResult } from './logs';
 export enum Workflow {
   GENERIC = 'generic',
   MANAGED = 'managed',
+  CI = 'ci',
```

Comment:
Maybe: 
```suggestion
  AUTO = 'auto',
```
or
```suggestion
  DETECT = 'detect',
```

I don't like `CI` in this context. What did you mean by `ci` in this context? EAS Build, GitHub actions, or sth else?